tests/languages/web/index.html

```diff
@@ -1,202 +1,204 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-  <meta charset="UTF-8" />
-  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <script src="./dist/iife/sdk.js"></script>
-  <script src="https://cdn.jsdelivr.net/npm/js-md5@0.7.3/build/md5.min.js"></script>
-  <title>Appwrite</title>
-</head>
-
-<body>
-  <p>File:</p>
-  <input type="file" id="file" />
-  <p>Large file: (over 5MB)</p>
-  <input type="file" id="file2" />
-  <button id="start">Start</button>
-  <script type="module">
-    document.getElementById("start").addEventListener("click", async () => {
-      let response;
-      let responseRealtime = "Realtime failed!";
-      // Init SDK
-      const {
-        Client,
-        Foo,
-        Bar,
-        General,
-        Query,
-        Permission,
-        Role,
-        ID,
-        MockType,
-        Payload,
-      } = Appwrite;
-      const client = new Client();
-
-      const foo = new Foo(client);
-      const bar = new Bar(client);
-      const general = new General(client);
-
-      client.setProject("console");
-      client.setEndpointRealtime("ws://demo.appwrite.io/v1");
-
-      client.subscribe("tests", (event) => {
-        responseRealtime = event.payload.response;
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
Lets revert formatting changes please in index.html

Reply:
Done